src/Extension/MemberExtension.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace SilverStripe\MFA\Extension;
+
+use SilverStripe\MFA\Model\RegisteredMethod;
+use SilverStripe\ORM\DataExtension;
+
+/**
+ * Extend Member to add relationship to registered methods and track some specific preferences
+ *
+ * @method RegisteredMethod[] RegisteredMethods
+ * @property string DefaultRegisteredMethod
+ */
+class MemberExtension extends DataExtension
+{
+    private static $has_many = [
+        'RegisteredMethods' => RegisteredMethod::class,
```

Comment:
```suggestion
        'RegisteredMFAMethods' => RegisteredMethod::class,
```

Suggesting this as I think it makes sense to add some more context to a MemberExtension.